src/modules/datepicker/directives/datepicker.directive.ts

```diff
@@ -12,6 +11,7 @@ import { IDatepickerLocaleValues, RecursivePartial, SuiLocalizationService } fro
 import { SuiPopupComponentController, PopupAfterOpen, PopupConfig, PopupTrigger } from "../../popup";
 import { SuiDatepicker, DatepickerMode } from "../components/datepicker";
 import { CalendarConfig, YearConfig, MonthConfig, DatetimeConfig, TimeConfig, DateConfig } from "../classes/calendar-config";
+import { isValid } from "date-fns";
```

Comment:
So the unused imports option is currently disabled because it's super unreliable... So just trying to delete things when I see them really until the analyzer is fixed.